embedding/cpp/flutter_app.cc

```diff
@@ -0,0 +1,199 @@
+// Copyright 2021 Samsung Electronics Co., Ltd. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "include/flutter_app.h"
+
+#include <system_info.h>
+
+#include <cassert>
+#include <cerrno>
+#include <fstream>
+
+#include "tizen_log.h"
+
+bool FlutterApp::OnCreate() {
+  TizenLog::Debug("Launching a Flutter application...");
+
+  FlutterWindowProperties window_prop = {};
+  window_prop.x = 0;
+  window_prop.y = 0;
+  if (system_info_get_platform_int("http://tizen.org/feature/screen.width",
+                                   &window_prop.width) != 0 ||
+      system_info_get_platform_int("http://tizen.org/feature/screen.height",
+                                   &window_prop.height) != 0) {
+    TizenLog::Error("Could not obtain the screen size.");
+    return false;
+  }
+
+  std::string res_path;
+  {
+    auto path = app_get_resource_path();
```

Comment:
app_get_resource_path can return null pointer. I think it would be better to check the return value.

Reply:
Thanks!